packages/engine/Source/Scene/VoxelCylinderShape.js

```diff
@@ -59,40 +59,16 @@ function VoxelCylinderShape() {
   this.shapeTransform = new Matrix4();
 
   /**
-   * @type {number}
-   * @private
-   */
-  this._minimumRadius = VoxelCylinderShape.DefaultMinBounds.x;
-
-  /**
-   * @type {number}
+   * @type {Cartesian3}
```

Comment:
Even though it's private, let's document what each component of the cartesian corresponds to– angle, height, range.